### PR TITLE
chore(protobuf): update protobuf definitions (display type enum added)

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/protobuf/src/index.ts [hunk 1]
```diff
@@ -5,4 +5,4 @@ export * from './types';
 export { parseConfigure, createMessageFromName, createMessageFromType } from './utils';
 export * as MessagesSchema from './messages-schema';
 // It's problem to reexport enums when they are under MessagesSchema namespace, check packages/connect/src/types/device.ts
-export { DeviceModelInternal } from './messages-schema';
+export { DeviceModelInternal, DisplayRotation, Enum_DisplayRotation } from './messages-schema';
```
Comment (mroz22) [inline packages/protobuf/src/index.ts]: is this the way we want to export these? cc @martykan

Comment (martykan) [reply packages/protobuf/src/index.ts]: I don't know... `DeviceModelInternal` seems important enough to re-export separately, but I don't think it's the best approach for every single enum. 
If we want to re-export in `device.ts` something like this should work:
```
export type DisplayRotation = PROTO.DisplayRotation;
```

File: packages/connect/src/types/device.ts [hunk 2]
```diff
@@ -147,7 +147,7 @@ export type UnreadableDevice = BaseDevice & {
 export type Device = KnownDevice | UnknownDevice | UnreadableDevice;
 export type Features = PROTO.Features;
 
-export { DeviceModelInternal } from '@trezor/protobuf';
+export { DeviceModelInternal, Enum_DisplayRotation, DisplayRotation } from '@trezor/protobuf';
```
Comment (peter-sanderson) [inline packages/connect/src/types/device.ts]: the export of the underscored value seems odd, but if its intentional :shrug:

**Review by peter-sanderson — APPROVED** [review]

File: packages/suite/src/views/settings/SettingsDevice/DisplayRotation.tsx [hunk 3]
```diff
@@ -76,12 +79,12 @@ export const DisplayRotation = ({ isDeviceLocked }: DisplayRotationProps) => {
...
                     selectedOption={currentRotation ?? undefined}
                     options={DISPLAY_ROTATIONS}
-                    onChange={(value: number) => {
+                    onChange={(value: DisplayRotationEnum) => {
                         dispatch(applySettings({ display_rotation: value }));
                         analytics.report({
                             type: EventType.SettingsDeviceChangeOrientation,
                             payload: {
-                                value: value as RotationValue,
+                                value: Enum_DisplayRotation[value],
```
Comment (mroz22) [inline packages/suite/src/views/settings/SettingsDevice/DisplayRotation.tsx]: todo: utility

**Pull request merged**